### ISSUE TITLE
"ValueError: PyCapsule_GetPointer called with incorrect name" when importing pynapple

### ISSUE THREAD

**Issue by edmundchong** [PeyracheLab/pynapple#102]
Hello! I installed pynapple following the instructions (pip install), and get the following error message when trying to import pynapple:

`ValueError                                Traceback (most recent call last)
Input In [2], in <cell line: 3>()
      1 import numpy as np
      2 import pandas as pd
----> 3 import pynapple as nap

File ~\anaconda3\envs\pynapple\lib\site-packages\pynapple\__init__.py:3, in <module>
      1 from .core import *
      2 from .process import *
----> 3 from .io import *

File ~\anaconda3\envs\pynapple\lib\site-packages\pynapple\io\__init__.py:1, in <module>
----> 1 from .misc import *

File ~\anaconda3\envs\pynapple\lib\site-packages\pynapple\io\misc.py:9, in <module>
      3 """
      4 Various io functions
      5 
      6 @author: Guillaume Viejo
      7 """
      8 import os, sys
----> 9 from .neurosuite import NeuroSuite
     10 from .phy import Phy
     11 from .loader import BaseLoader

File ~\anaconda3\envs\pynapple\lib\site-packages\pynapple\io\neurosuite.py:16, in <module>
     14 import numpy as np
     15 from .. import core as nap
---> 16 from .loader import BaseLoader
     17 import pandas as pd
     18 from pynwb import NWBFile, NWBHDF5IO

File ~\anaconda3\envs\pynapple\lib\site-packages\pynapple\io\loader.py:15, in <module>
     13 import numpy as np
     14 from .. import core as nap
---> 15 from .loader_gui import BaseLoaderGUI
     17 from PyQt5.QtWidgets import QApplication
     19 from pynwb import NWBFile, NWBHDF5IO, TimeSeries

File ~\anaconda3\envs\pynapple\lib\site-packages\pynapple\io\loader_gui.py:10, in <module>
      8 import scipy.signal
      9 import numpy as np
---> 10 from pyqtgraph import PlotWidget, plot
     11 import pyqtgraph as pg
     13 class TTLDetection(QDialog):

File ~\anaconda3\envs\pynapple\lib\site-packages\pyqtgraph\__init__.py:17, in <module>
     13 import numpy  # # pyqtgraph requires numpy
     15 ## 'Qt' is a local module; it is intended mainly to cover up the differences
     16 ## between PyQt and PySide.
---> 17 from .colors import palette
     18 from .Qt import QtCore, QtGui, QtWidgets
     19 from .Qt import exec_ as exec

File ~\anaconda3\envs\pynapple\lib\site-packages\pyqtgraph\colors\palette.py:1, in <module>
----> 1 from ..Qt import QtGui
      4 def getQDarkStyleDarkQPalette():
      5     BG_DARK = QtGui.QColor('#19232D')

File ~\anaconda3\envs\pynapple\lib\site-packages\pyqtgraph\Qt\__init__.py:170, in <module>
    167 from PyQt5 import uic
    169 try:
--> 170     from PyQt5 import QtSvg
    171 except ImportError as err:
    172     QtSvg = FailedImport(err)

ValueError: PyCapsule_GetPointer called with incorrect name`

**Comment by apeyrache** [comment]
Hi Edmond!
Seems like an error with Qt. Just to make sure, did you pop install in a new Vonda environment? There are issues with various versions of Qt and it is possible it is incompatible with the one you have already installed.

**Comment by gviejo** [comment]
Hi Edmond,

What is your python version?

**Comment by edmundchong** [comment]
Hi Guillaume,

It is 3.10.4.

Thanks!

**Comment by edmundchong** [comment]
Hi Guillaume,

I found the issue--
I have to use pip to install pyqt5 (giving me version 5.15.7 as opposed to 5.9.2 above). 
And after that, I have to be careful about installing any other packages which use pyqt--for example, if I do `conda install jupyter`, or `conda install matplotlib`, these will override pyqt with 5.9.2 and then pynapple fails to load. I had to install these other packages with the following:

```
conda install -c conda-forge notebook
pip install matplotlib
```

Things seem to work now, and I managed to run your example dataset. Not too sure why 5.9.2 works for you as per our separate email conversation, but as you suggested, weird conflicts arise between pip/conda. Thanks again!
